source/application/components/application/index.js

```diff
@@ -118,15 +118,15 @@ Application.propTypes = {
 	themeLoading: t.bool.isRequired
 };
 
-function createLinks(styles, options) {
-	return styles.map(createStyle(options));
+function createLinks(styles) {
+	return styles.map(createStyle());
 }
 
-function createStyle(options) {
+function createStyle() {
 	return style => {
 		return {
 			'rel': 'stylesheet',
-			'href': `${options.base}/style/${style}.css`,
+			'href': `/style/${style}.css`,
```

Comment:
This should work for scenarios where patternplate is deployed on subpaths, e.g.

`http://some.tld/some-path/`

The proposed solution will break for the url above. We should use `options.base` to compute the relative path from the current url to the `style/${style}.css`.